### PR TITLE
Create Privacy section in the guide

### PR DESCRIPTION
**Pull request by vacekj**
Starting this as a place to document the privacy features and invariants of the protocol. An example would be that trades are private as long as there are at least two in a batch.

This should serve as a one-stop point for _regular users_ (not developers, node operators or other technical users of the protocol) to learn about what actions are private, and under what condition.

### CODE REVIEW

**Comment by TalDerei** [comment]
reference https://github.com/penumbra-zone/penumbra/issues/3867 / https://github.com/penumbra-zone/penumbra/issues/3997 in the protocol docs

**Comment by vacekj** [comment]
@TalDerei and others to whom it may concern - I've described the transfers and swaps privacy properties in this PR, since that's what I believe most users will care about. I've dropped a TODO in the other sections on staking etc. One other area which I think we should document prior to it being enabled are IBC transfers - I'm not sure what the privacy properties are there, maybe @grod220 you can help in this regard since we talked about the private IBC inbound transfers. 

I would propose we merge this now (if all the info and wording is correct) and continue filling this in as we go.

File: docs/guide/src/privacy.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Transfers
+
+Transfers do not reveal the asset, amount, sender or recipient identity. Transfers are also unlinkable to each other.
+
+## Swaps
+
+Initiating a swap does not reveal the identity of the swapper or the pre-paid claim fee, but it does reveal the assets and amounts in the swap.
+
+Claiming a swap does not reveal the amounts, asset types or the identity of the claimant.
```
Comment (redshiftzero) [inline docs/guide/src/privacy.md:13]: While the two notes that get minted in a successful `SwapClaim` are private (including amounts/asset types), the `SwapClaim` action does reveal the _trading pair_ due to the claimant having to provide the public `BatchSwapOutputData` which has the trading pair as a field. 

The shielded output notes are not linkable to the `BatchSwapOutputData` though, so an observer of the chain just learns an anonymous someone minted shielded outputs of a swap of assets A/B from a swap that was executed in block X.

File: docs/guide/src/privacy.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### Conditions
+
+The above holds true as long as there are at least two swaps in a batch.
+
+## Governance Voting
+
+During a vote, the voting power (amount and asset type of the staked note that's used for voting),
```
Comment (redshiftzero) [inline docs/guide/src/privacy.md:21]: You could optionally add that the identity of the validator that the voter has delegated to is revealed in a delegator vote. It's implicit given you mention that the asset type of the staked note is revealed, but it's possibly more understandable for end users as the identity of their validator.

File: docs/guide/src/privacy.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Swaps
+
+Initiating a swap does not reveal the identity of the swapper or the pre-paid claim fee, but it does reveal the assets and amounts in the swap.
+
+Claiming a swap does not reveal the amounts, asset types or the identity of the claimant.
+
+### Conditions
+
+The above holds true as long as there are at least two swaps in a batch.
```
[the remark this answers is not in the document]
Comment (vacekj) [reply docs/guide/src/privacy.md:17]: Thank you, fixed.

File: docs/guide/src/privacy.md (hunk 1 above)
Comment (vacekj) [reply docs/guide/src/privacy.md:13]: Thank you - I've tried to communicate this better in the updated version.

File: docs/guide/src/privacy.md (hunk 2 above)
Comment (vacekj) [reply docs/guide/src/privacy.md:21]: Good point - added that in with a note. Feel free to reword or Resolve if good.

**Review by redshiftzero — APPROVED** [review]

**Pull request merged**